templates/focus-ios.xliff

```diff
@@ -547,10 +571,62 @@
         <source>Open in %@</source>
         <note>Text for the share menu option when a user wants to open a page in Focus.</note>
       </trans-unit>
+      <trans-unit id="ShareMenu.PageActions">
+        <source>Page Actions</source>
+        <note>Title for the share menu where users can take actions for the current website they are on.</note>
+      </trans-unit>
       <trans-unit id="ShareMenu.RequestDesktop">
         <source>Request Desktop Site</source>
         <note>Text for the share menu option when a user wants to reload the site as a desktop</note>
       </trans-unit>
+      <trans-unit id="ShareMenu.ShareOpenChrome">
+        <source>Open in Chrome</source>
+        <note>Text for the share menu option when a user wants to open the current website in the Chrome app.</note>
+      </trans-unit>
+      <trans-unit id="ShareMenu.ShareOpenFirefox">
+        <source>Open in Firefox</source>
+        <note>Text for the share menu option when a user wants to open the current website in the Firefox app.</note>
+      </trans-unit>
+      <trans-unit id="ShareMenu.ShareOpenSafari">
+        <source>Open in Safari</source>
+        <note>Text for the share menu option when a user wants to open the current website in the Safari app.</note>
+      </trans-unit>
+      <trans-unit id="ShareMenu.SharePage">
+        <source>Share Page With...</source>
+        <note>Text for the share menu option when a user wants to share the current website they are on through another app.</note>
+      </trans-unit>
+      <trans-unit id="Siri.add">
+        <source>Add</source>
+        <note>Button to add a favorite URL to Siri.</note>
+      </trans-unit>
+      <trans-unit id="Siri.addTo">
+        <source>Add to Siri</source>
+        <note>Button to add a specified shortcut option to Siri.</note>
+      </trans-unit>
+      <trans-unit id="Siri.editOpenUrl">
+        <source>Re-Record or Delete Shortcut</source>
+        <note>Label for button to edit the Siri phrase or delete the Siri functionality.</note>
+      </trans-unit>
+      <trans-unit id="Siri.erase">
+        <source>Erase</source>
+        <note>Title of option in settings to set up Siri to erase</note>
+      </trans-unit>
+      <trans-unit id="Siri.eraseAndOpen">
+        <source>Erase &amp; Open</source>
+        <note>Title of option in settings to set up Siri to erase and then open Focus/Klar. %@ is the application name.</note>
```

Comment:
Seems like the last part of the comment can be deleted, since there is no "%@" in the string? Unless it's missing from the string? :)